ThingIFSDK/ThingIFSDK/Command.swift

```diff
@@ -91,61 +83,44 @@ open class Command: NSObject, NSCoding {
     internal init(commandID: String?,
          targetID: TypedID,
          issuerID: TypedID,
-         schemaName: String,
-         schemaVersion: Int,
-         actions:[Dictionary<String, Any>],
-         actionResults:[Dictionary<String, Any>]?,
+         actions: [(alias: Alias, actions: [String : Any])],
+         actionResults: [(alias: Alias, actions: [String : Any])]?,
          commandState: CommandState?,
          firedByTriggerID: String? = nil,
          created: Date? = nil,
          modified: Date? = nil,
          title: String? = nil,
          commandDescription: String? = nil,
          metadata: Dictionary<String, Any>? = nil) {
-        if commandID != nil {
-            self.commandID = commandID!
-        }else {
-            self.commandID = ""
-        }
+        self.commandID = commandID ?? ""
```

Comment:
@yongpingchen 

It looks good. Moreover, We can make `self.command` non nil and non empty.

`Command` class represents a command created in server. The command must have command id.

I will change `commandID` argument to non optional.